layouts/partials/rbac-permissions-table.html

```diff
@@ -15,7 +15,7 @@
         {{- if ne .attributes.name nil -}}
         <tr>
           <td>
-            <code>{{- .attributes.name -}}</code>
+            {{- .attributes.display_name -}}<br>(<code>{{- .attributes.name -}}</code>)
```

Comment:
Tested on the [preview site](https://docs-staging.datadoghq.com/julia.gu/table-layout/account_management/rbac/permissions/#permissions-list) and it looks great! I believe @wangwillson1 had raised a concern that it might look crowded if we added another column but the parentheses solution is a nice simple one.